apps/prairielearn/src/pages/instructorQuestionSettings/instructorQuestionSettings.ejs

```diff
@@ -100,7 +100,7 @@
                       <span class="badge color-gray1"><%= sharing_set.name %></span>
                     <% }); %>
                   <% } %>
-                  <% if (authz_data.has_course_permission_own && !course.example_course) { %>
+                  <% if (authz_data.has_course_permission_own) { %>
```

Comment:
Is this the key change in this file? To allow owners of the example course to turn on sharing?

Reply:
Correct; the other ones were just opportunistic formatting tweaks. Without this, there's no way for anyone to actually mark an example course question as shared.

Reply:
Perfect, thanks!

Reply:
I went to go check if we needed to make this same change on the backend in `instructorQuestionSettings.js`, but I guess we never had that check there (oops).